tests/test_atom_client.py

```diff
@@ -0,0 +1,335 @@
+# -*- coding: UTF-8 -*-
+import os
+import pytest
```

Comment:
There should be a blank link before this import to separate std libs from 3rd-party libs.